envi/archs/arm/__init__.py

```diff
@@ -52,8 +50,6 @@ class ThumbModule(envi.ArchitectureModule):
     '''
     This architecture module will *not* shift to ARM mode.  Evar.
     '''
-    #Current ARM version working in - Not fully integrated yet
-    archVersion = ('ARMv7A')
 
     def __init__(self, name='ARMv7A'):
         self.archVersion = name
```

Comment:
please don't store this twice.  it's handed into envi.ArchitectureModule.__init__() already, which stores it.